modules/hmftools/1.0/hmftools.smk

```diff
@@ -70,16 +90,16 @@ rule _hmftools_input_bam:
         bam = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam", 
         bai = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bai", 
     run:
-        op.relative_symlink(input.bam, output.bam)
-        op.relative_symlink(input.bai, output.bai)
+        op.absolute_symlink(input.bam, output.bam)
```

Comment:
I can't comment on unchanged code but I noticed this module still has a `VERSION_MAP` variable - could you change it to `VERSION_MAP_HMFTOOLS`?

Reply:
It also has global wildcard constraints. Could you remove those and add them to one of the later rules instead?

Reply:
I addressed both the `VERSION_MAP` and wildcards comments. Thanks!